apiserver/params/params.go

```diff
@@ -1371,3 +1371,21 @@ type UpgradeSeriesUnitsResult struct {
 	Error     *Error   `json:"error,omitempty"`
 	UnitNames []string `json:"unit-names"`
 }
+
+// LXDProfileResult contains the charm.LXDProfile information in addition to
+// the name of the profile.
+type LXDProfileResult struct {
```

Comment:
Normally we don't add yaml serialisation directives in the params package.

If you need to serialse, you should have a different structure in the api package.